### PR TITLE
25.0 Final Changes

### PR DESCRIPTION
**Pull request by fanquake**
Final changes for v25.0.
PR for bitcoincore.org is here: https://github.com/bitcoin-core/bitcoincore.org/pull/970.

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -1,2 +1,332 @@
...
+- Leonardo Araujo
+- Leonardo Lazzaro
+- Luke Dashjr
+- MacroFake
+- MarcoFalke
+- Martin Leitner-Ankerl
+- Martin Zumsande
+- Matt Whitlock
+- Matthew Zipkin
+- merge-script
```
Comment (josibake) [inline doc/release-notes.md:294]: lol credit where credit is due, i guess?

**Review by josibake — COMMENTED** [review]
ACK https://github.com/bitcoin/bitcoin/pull/27686/commits/9ded442dffb827d8762c0becce9d10fa283d1a3a

looks good, although we probably don't need to credit `merge-script` as a contributor

**Comment by josibake** [comment]
reACK https://github.com/bitcoin/bitcoin/pull/27686/commits/6675cf809543866792bf6706372fc9ac253d3e99

File: doc/release-notes.md [hunk 2]
```diff
@@ -1,2 +1,331 @@
...
+- A new `/rest/deploymentinfo` endpoint has been added for fetching various
+  state info regarding deployments of consensus changes. (#25412)
+
+Binary verification
+----
+
+- The binary verification script has been updated. In previous releases it
+  would verify that the binaries had been signed with a single "release key".
+  In this release and moving forward it will verify that the binaries are
+  signed by a _threshold of trusted keys_. For more detairs and
```
Comment (vostrnad) [inline doc/release-notes.md:200]:
```suggestion
  signed by a _threshold of trusted keys_. For more details and
```

Comment (fanquake) [reply doc/release-notes.md:200]: Thanks. Fixed.

**Review by theStack — COMMENTED** [review]
Release notes for the new `scanblocks` RPC (https://github.com/bitcoin/bitcoin/pull/23549) are currently missing, could take what @kouloumos prepared in the wiki: https://github.com/bitcoin-core/bitcoin-devwiki/wiki/scanblocks-RPC-release-notes (https://github.com/bitcoin/bitcoin/pull/23549#issuecomment-1277753723).

**Comment by fanquake** [comment]
> Release notes for the new scanblocks RPC (https://github.com/bitcoin/bitcoin/pull/23549) are currently missing,

**Review by Jones098 — APPROVED** [review]

**Pull request merged**